frontend/src/components/ui/data-grid/controllers/rows.ts

```diff
@@ -46,38 +56,32 @@ export class DataGridRowsController implements ReactiveController {
     }
   }
 
-  private setRowsFromItems<T extends GridItem = GridItem>(items: T[]) {
+  private setRowsFromItems(items: Item[]) {
     const rowKey = this.#host.rowKey;
 
     this.rows = new Map(
       this.#host.rowKey
```

Comment:
(following the previous comment)
```suggestion
      rowKey
```